genai-perf/tests/test_input_rankings.py

```diff
@@ -76,8 +77,8 @@ def test_get_input_dataset_from_rankings_file(self, mock_file):
             batch_size=batch_size,
             num_prompts=100,
         )
-        inputs = Inputs(config)
-        dataset = inputs._get_input_dataset_from_rankings_files(
+        irf = InputRetrieverFactory(config)
```

Comment:
Could you use a different variable name and keep it consistent with the other test file? Irf is unintuitive, especially if you see it later in the code.

Reply:
Updated to `input_retriever_factory`